bundles/framework/admin-announcements/view/AnnouncementsFormMapper.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import {AnnouncementsForm} from './AnnouncementsForm';
+import "antd/dist/antd.css";
+import PropTypes from 'prop-types';
+import { Button, Collapse } from "antd";
+import { PlusOutlined } from '@ant-design/icons';
+import { Controller, LocaleConsumer } from 'oskari-ui/util';
+import styled from 'styled-components';
+
+/*
+Maps all created announcements into a collapse.
+*/
+
+const { Panel } = Collapse;
+
+const StyledButton = styled(Button)`
+    margin-top: 10px;
+`;
+
+const AnnouncementsFormMapper = ({controller,  announcements, updated, title, activeKey }) => {
```

Comment:
Is this more like `AnnouncementList`? I don't understand the naming here. Can you give more info on why should this be called "FormMapper"?

Reply:
renamed